<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_vbox">
   <item>
    <widget class="QGroupBox" name="general_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="general_group_layout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_remove_nodes_and_branches">
        <property name="toolTip">
         <string>Select nodes or branches</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="node_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Node options</string>
     </property>
     <layout class="QFormLayout" name="node_layout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_node_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="node_name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="btn_insert_branch">
        <property name="toolTip">
         <string>Select one or two nodes</string>
        </property>
        <property name="text">
         <string>Insert branch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="branch_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Branch options</string>
     </property>
     <layout class="QFormLayout" name="branch_layout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_branch_weight">
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="branch_weight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="graph_operation_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Graphoperations</string>
     </property>
     <layout class="QGridLayout" name="graph_operation_layout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_chaining_rule">
        <property name="toolTip">
         <string>Select two subsequent branches</string>
        </property>
        <property name="text">
         <string>Chaining rule</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_combine_parallel">
        <property name="toolTip">
         <string>Select two parallel branches</string>
        </property>
        <property name="text">
         <string>Combine parallel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_eliminate_node">
        <property name="toolTip">
         <string>Select one node which can be eliminated</string>
        </property>
        <property name="text">
         <string>Eliminate node</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_eliminate_self_loop">
        <property name="toolTip">
         <string>Select a branch which is a self-loop</string>
        </property>
        <property name="text">
         <string>Eliminate self-loop</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_graph_transposition">
        <property name="text">
         <string>Graph transposition</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_scale_path">
        <property name="toolTip">
         <string>Select one or multiple nodes</string>
        </property>
        <property name="text">
         <string>Scale path</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_invert_path">
        <property name="toolTip">
         <string>Select a branch that has a start node with zero ingoing branches</string>
        </property>
        <property name="text">
         <string>Invert path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="export_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Exports</string>
     </property>
     <layout class="QGridLayout" name="export_layout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_generate_mason">
        <property name="toolTip">
         <string>Select a start and a end node</string>
        </property>
        <property name="text">
         <string>Generate Mason</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_generate_tikz">
        <property name="text">
         <string>Generate TikZ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
